webpack.config.js

```diff
@@ -1,6 +1,6 @@
 var path = require('path'),
     Webpack = require('webpack'),
-    ExtractTextPlugin = require('extract-text-webpack-plugin');
+    MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
Updated configuration in Webpack config to use `mini-css-extract-plugin` instead of `extract-text-webpack-plugin`. [reference](https://dev.mobify.com/v1.x/how-to-guides/categories/upgrades/webpack-4)